Sources/SwiftSyntax/generated/SyntaxBaseNodes.swift

```diff
@@ -474,6 +474,7 @@ public extension Syntax {
 /// - ``DeclReferenceExprSyntax``
 /// - ``DictionaryExprSyntax``
 /// - ``DiscardAssignmentExprSyntax``
+/// - ``DoExprSyntax``
```

Comment:
Ideally we wouldn’t include experimental nodes here.

Reply:
Good catch!